dev/tasks/conda-recipes/arrow-cpp/activate.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# for the gdb-wrappers, we need to create a symlink that
+# contains the full path of the lib _within_ the installed
+# env, which we don't have until the env is created.
+
+# doesn't come with a deactivate script, because the symlink
+# is benign and doesn't need to be deleted.
+
+# where the GDB wrappers get installed
+GDB_PREFIX=$CONDA_PREFIX/share/gdb/auto-load
+
+# If the directory is not writable, nothing can be done
+if [ ! -w $GDB_PREFIX ]; then
+    return
+fi
+
+# this needs to be in sync with the respective patch
+PLACEHOLDER=replace_this_section_with_absolute_slashed_path_to_CONDA_PREFIX
```

Comment:
Ah, this won't work unless we carry something like https://github.com/conda-forge/arrow-cpp-feedstock/blob/main/recipe/patches/0001-don-t-bake-non-relocatable-CMAKE_INSTALL_FULL_LIBDIR.patch here... It could be done just within `dev/tasks/conda-recipes`, depends a bit if people want that.

The default install doesn't work with the conda-forge infrastructure because it bakes the wrong PREFIX (the one that's there while the package is being created, rather than installed) into the gdb path. For details see https://github.com/conda-forge/arrow-cpp-feedstock/issues/935